AAP-585 Importing CA certificates for automation mesh (#383)
RedHatInsights/red-hat-ansible-automation-platform-documentation: downstream/modules/platform/con-install-mesh.adoc

[id="con-install-mesh_{context}"]

> = Installing {AutomationMesh}

Reviewer: ```suggestion
= {AutomationMesh} installation 
```

> You can install {AutomationMesh} using the {PlatformNameShort} installer found here. The installer contains an `inventory` file where you can define the nodes, groups, and peer relationships that make up your mesh network. Users looking to upgrade to {AutomationMesh} can run the latest version of the {PlatformNameShort} installer after configuring their {AutomationMesh} network on the `inventory` file.

Reviewer: ```suggestion
You use the {PlatformNameShort} installation program to set up {AutomationMesh} or to upgrade to {AutomationMesh}.
To provide {PlatformNameShort} with details about the nodes, groups, and peer relationships in your mesh network, you define them in an the `inventory` file in the installer bundle.
```

> .Additional Resources

Reviewer: ```suggestion
[role="_additional-resources"]
.Additional resources
```

* Learn how to use the {PlatformNameShort} installer by reviewing the link:https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.1/html/red_hat_ansible_automation_platform_installation_guide/index[{PlatformName} Installation Guide].

> * Learn about the ways you can design your {AutomationMesh} network by reviewing <<assembly-example-topologies, Automation Mesh Design Patterns.>>

Reviewer: Need to update the xref to the Design Patterns assembly. This is another good reason for putting only the link and link text in the additional resources section. It's a lot easier to see what those links are and maintain them.

Speaking of which, should that be a `2.1` URL or `2.2` for the installation guide?

> * Learn about the ways you can design your {AutomationMesh} network by reviewing <<assembly-example-topologies, Automation Mesh Design Patterns.>>

Author: Ah, link should include {PlatformVers} where the version number is so that it links to the selected version. Good catch
